source/cloud/aws/sagemaker.md

```diff
@@ -35,7 +35,11 @@ set -e
 
 sudo -u ec2-user -i <<'EOF'
 
-mamba create -y -n rapids {{ rapids_conda_channels }} {{ rapids_conda_packages }} ipykernel
+mamba create -y -n rapids {{ rapids_conda_channels }} {{ rapids_conda_packages }} \
+    boto3 \
+    ipykernel \
+    sagemaker
```

Comment:
The document this is referneced from recommends trying out the `rapids-sagemaker-higgs` notebook, which begins by importing `boto3` and `sagemaker`. Those libraries are not shipped in the `rapidsai/core` Docker image used to run the notebook instance.

This ensures that they're installed ahead of time.